api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -73,7 +73,7 @@ def get_updated_content(content: str) -> str:
             return content.strip()
 
         # Get the corresponding saved readme output path based on the zip filename
-        readme_filename = re.sub(r"^[A-Z\d]+_", "", Path(zip_file.filename).stem) + ".md"
+        readme_filename = re.sub(r"^[A-Z]{5}\d{6}_", "", Path(zip_file.filename).stem) + ".md"
```

Comment:
I've made the regex pattern more explicit for `PROJECTID` (e.g., `SCPCP999990`) and adjusted this file as well.